packages/__docs__/src/App/index.tsx

```diff
@@ -824,6 +821,9 @@ class App extends Component<AppProps, AppState> {
               <div css={this.props.styles?.hamburger}>
                 <InstUISettingsProvider>
                   <IconButton
+                    themeOverride={{
+                      secondaryBorderColor: '#343434'
+                    }}
```

Comment:
Now the hamburger menu is overlapping the content, so I made its border have a bit more contrast

Reply:
They dont have to agree on smaller changes in our docs app :)